server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -229,13 +238,57 @@ export function create(
 		},
 	);
 
+	/**
+	 * Delete a document
+	 */
+	router.delete(
+		"/:tenantId/document/:id",
+		validateRequestParams("tenantId", "id"),
+		validateTokenClaimsScopes(DocDeleteScopeType),
+		verifyStorageToken(tenantManager, config, tokenManager),
+		throttle(generalTenantThrottler, winston, tenantThrottleOptions),
```

Comment:
>throttle(generalTenantThrottler, winston, tenantThrottleOptions), [](http://example.com/codeflow?start=2&length=65)

We may need to specify throttling limits. Joel, Khushboo, and Zach know the details.

Reply:
Talk with Joel, will update throttling config in next PR

Reply:
Removed throttle config in this PR